source/includes/fact-compressors-option-desc.rst

```diff
@@ -0,0 +1,14 @@
+Specify the compression algorithm used between the MongoDB server and
```

Comment:
(parallelism) 
Consider "Specifies" rather than "Specify" to create parallelism between this and the other options.